skore/src/skore/ui/project_routes.py

```diff
@@ -51,6 +56,111 @@ class SerializableProject:
     views: dict[str, Layout]
 
 
+def __cross_validation_item_as_serializable(item: CrossValidationItem) -> dict:
+    # Get tabular results (the cv results in a dataframe-like structure)
+    cv_results = copy.deepcopy(item.cv_results_serialized)
+    cv_results.pop("indices", None)
+
+    tabular_results = {
+        "name": "Cross validation results",
+        "columns": list(cv_results.keys()),
+        "data": list(zip(*cv_results.values())),
+    }
+
+    # Get scalar results (summary statistics of the cv results)
+    def metric_title(metric):
+        m = metric.replace("_", " ")
+        title = f"Mean {m}"
+        if title.endswith(" time"):
+            title = title + " (seconds)"
+        return title
```

Comment:
Can you leave this function outside of the function? So it can be defined only one-time by python in the scope of the module.

Reply:
See #971